Addition of GPU_CORE_FREQUENCY_STEP for Intel GPUs (#2724)
geopm/geopm: service/docs/source/geopm_pio_levelzero.7.rst

    *  **Format**: double
    *  **Unit**: hertz

``LEVELZERO::GPU_CORE_FREQUENCY_STEP``

>     The GPU Compute Hardware frequency step size in hertz.  If the step size is variable the average of all steps is provided.

Author: ```suggestion
    The GPU Compute Hardware frequency step size in hertz.  The average step size is provided in the case where the step size is variable.
```
